### PR TITLE
 ticdc: adjust TOC and reorganize troubleshooting & faq

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9945, https://github.com/pingcap/docs-cn/pull/9845
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by overvenus — APPROVED** [review]

File: TOC.md [hunk 1]
```diff
@@ -485,14 +485,17 @@
     - [Overview](/ticdc/ticdc-overview.md)
     - [Deploy](/ticdc/deploy-ticdc.md)
     - [Maintain](/ticdc/manage-ticdc.md)
+    - Monitor and Alert
+      - [Monitor](/ticdc/monitor-ticdc.md)
```
Comment (lilin90) [inline TOC.md:489]: "Monitor" sounds like a doc of how-to steps, which is not in this case.
```suggestion
      - [Monitoring Metrics](/ticdc/monitor-ticdc.md)
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
+---
+title: FAQs
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:2]:
```suggestion
title: TiCDC FAQs
```

File: ticdc/ticdc-faq.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+summary: Learn the FAQs you might encounter when you use TiCDC.
+---
+
+# FAQs
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:6]:
```suggestion
# TiCDC FAQs
```

**Review by lilin90 — APPROVED** [review]
LGTM

File: migrate-from-tidb-to-tidb.md [hunk 4]
```diff
@@ -238,7 +238,7 @@ After setting up the environment, you can use the backup and restore functions o
 
 3. Enable GC.
 
-    In incremental migration using TiCDC, GC only removes history data that is replicated. Therefore, after creating a changefeed, you need to run the following command to enable GC. For details, see [What is the complete behavior of TiCDC garbage collection (GC) safepoint?](/ticdc/troubleshoot-ticdc.md#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint).
+    In incremental migration using TiCDC, GC only removes history data that is replicated. Therefore, after creating a changefeed, you need to run the following command to enable GC. For details, see [What is the complete behavior of TiCDC garbage collection (GC) safepoint?](/ticdc/ticdc-faq.md#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint).
```
Comment (shichun-0415) [inline migrate-from-tidb-to-tidb.md:241]:
```suggestion
    In incremental migration using TiCDC, GC only removes history data that is replicated. Therefore, after creating a changefeed, you need to run the following command to enable GC. For details, see [What is the complete behavior of TiCDC garbage collection (GC) safepoint?](/ticdc/ticdc-faq.md#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint). 
```

**Pull request merged**